src/features/instance/operations/queries/getSearchByValue.ts

```diff
@@ -44,9 +45,20 @@ function getSearchByValueOptions({
 	};
 }) {
 	return queryOptions({
-		queryKey: [instanceId, 'search_by_value'] as const,
+		queryKey: [
+			'search_by_value',
+			hash_attribute,
+			instanceId,
+			pagination.pageIndex || 0,
+			pagination.pageSize || 0,
+			schemaName,
+			sortTableDataParams.attribute || 'default',
+			sortTableDataParams.descending || false,
+			tableName,
```

Comment:
Adding these here allows us to fetch whenever these change, instead of having to trigger the refetch manually.